scout/server/blueprints/dashboard/controllers.py

```diff
@@ -23,130 +23,143 @@ def get_dashboard_info(adapter, institute_id=None, slice_query=None):
+    # reflect verified variants in all available cases for an institute
+    general_info = get_general_case_info(adapter, institute_id=institute_id)
+    total_cases = general_info['total_cases']
+    sliced_case_ids = general_sliced_info['case_ids']
+    verified_query = {
+        'verb' : 'validate',
+    }
+    if institute_id: # filter by institute if users wishes so
+        verified_query['institute'] =  institute_id
+
+    # Case level information
+    sliced_validation_cases = set()
+    sliced_validated_cases = set()
+
+    # Variant level information
+    validated_tp = set()
+    validated_fp = set()
+    var_valid_orders = 0 # use this counter to count 'True Positive', 'False positive' and 'Not validated' vars
+
+    validate_events = adapter.event_collection.find(verified_query)
+    for validate_event in list(validate_events):
+        case_id = validate_event.get('case')
+        var_obj = adapter.variant(case_id=case_id, document_id=validate_event['variant_id'])
+
+        if var_obj: # Don't take into account variants which have been removed from db
+            var_valid_orders += 1
+            if case_id in sliced_case_ids:
+                sliced_validation_cases.add(case_id) # add to the set. Can't add same id twice since it'a a set
+
+            validation = var_obj.get('validation')
+            if validation and validation in ['True positive', 'False positive']:
+                if case_id in sliced_case_ids:
+                    sliced_validated_cases.add(case_id)
+                if validation == 'True positive':
+                    validated_tp.add(var_obj['_id'])
+                elif validation == 'False positive':
+                    validated_fp.add(var_obj['_id'])
+
+    n_validation_cases = len(sliced_validation_cases)
+    n_validated_cases = len(sliced_validated_cases)
+
+    # append
+    overview.append(
+    {
+        'title': 'Validation ordered',
+        'count': n_validation_cases,
+        'percent': n_validation_cases / total_sliced_cases,
+    })
+    overview.append(
+    {
+        'title': 'Validated cases',
+        'count': n_validated_cases,
+        'percent': n_validated_cases / total_sliced_cases,
+    })
 
     data['overview'] = overview
 
     variants = []
     nr_validated = len(validated_tp) + len(validated_fp)
-    if nr_ordered:
-        variants.append(
-            {
-                'title': 'Validation ordered',
-                'count': nr_ordered,
-                'percent': 1
+    variants.append(
+        {
+            'title': 'Validation ordered',
+            'count': var_valid_orders,
+            'percent': 1
+        }
+    )
+
+    # taking into account that var_valid_orders might be 0:
+    percent_validable = 0
+    percent_validated = 0
+
+    if var_valid_orders:
+        percent_validable = len(validated_tp) / var_valid_orders
+        percent_validated = len(validated_fp) / var_valid_orders
```

Comment:
Could we change the name of these? percent_validated_tp, percent_validated_fp perhaps?

Reply:
Sure, didn't realize how bad I named them